### PR TITLE
Bionano settings commented out by default in scout settings

### PR DESCRIPTION
**Pull request by northwestwitch**
This PR adds a functionality or fixes a bug:
- Do not launch the Bionano extension if these settings are not customised

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Show case default panel on caseS page
 - CADD filter for Cancer Somatic SNV variantS - show score
 - SpliceAI-lookup link (BROAD, shows SpliceAI and Pangolin) from variant page
-- BioNano Access server API - check projects, samples and fetch FSHD reports
+- BioNano Access server API - check projects, samples and fetch FSHD reports.
```
Comment (dnil) [inline CHANGELOG.md:16]: 😄

File: CHANGELOG.md [hunk 2]
```diff
@@ -24,6 +24,7 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Visualization of cancer variants where cancer case has no affected individual
 - ProteinPaint gene link (small StJude API change)
 - Causative MEI variant link on causatives page
+- Bionano access api settings commented out by default in Scout config file
```
Comment (dnil) [inline CHANGELOG.md:27]:
```suggestion
- Bionano access api settings commented out by default in Scout demo config file.
```
😉

File: scout/server/config.py [hunk 3]
```diff
@@ -129,6 +129,6 @@
 SV_RANK_MODEL_LINK_POSTFIX = "-.ini"
 
 # BioNano Access Server extension
-BIONANO_ACCESS = "https://bionano-access.scilifelab.se"
-BIONANO_USERNAME = "USERNAME"
-BIONANO_PASSWORD = "PASSWORD"
+# BIONANO_ACCESS = "https://bionano-access.scilifelab.se"
```
Comment (dnil) [inline scout/server/config.py:132]: 💯

**Review by dnil — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (northwestwitch) [reply CHANGELOG.md:16]: Just a typo, it was not intended :)

**Pull request merged**